layout/_layout.njk

```diff
@@ -3,9 +3,12 @@
 <head>
   {{ partial('_partials/head/head.njk', {}, {cache: theme.cache.enable}) }}
   {%- include '_partials/head/head-unique.njk' -%}
+  <script{{ pjax }} src="{{ url_for(theme.js) }}/load-config.js"></script>
```

Comment:
Why not use `next_js` function?

Reply:
I searched and found NexT prefers to use `<script{{ pjax }} >`. There are only one line in `schedule.njk` using `next_js('schedule.js', true)`. And I found `next_js` will add `data_pjax` once its second parameter is `true`, on matter pjax is enabled or not.

But, as you reply, I realized that NexT is happen to be switching to use `next_js`. (right?)
Will do in a commit later.

Reply:
`next_js` helper does not have a pjax option because these scripts are designed to load only once.
You can try to register a `pjax:success` event in the script, instead of using `data-pjax` attribute, e.g.

https://github.com/next-theme/hexo-theme-next/blob/eca3ed344374af0deacfc9494d02a001f8ebf0bd/source/js/local-search.js#L300-L303

Reply:
I think there should be a function somewhere, `NexT.utils.addLoadListener` for example, it recieves a function, and calls the function instantly **and** attach to `pjax:success` event.

Like a `NexT` version `window.onload`.

It is not relevant with this pull request, but a suggestion to the future djax related developing.